### PR TITLE
ProtectionLevelName parameter is missing

### PR DESCRIPTION
**Pull request by PowershellScripts**
Added missing parameter and its description

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 1]
```diff
@@ -843,6 +843,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+
+### -ProtectionLevelName
+
+Policy tag.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:849]:
```suggestion
The conditional access protection level name. For example "urn:microsoft:req1"
```

**Review by get-itips — APPROVED** [review]
Thanks for your contribution, please see my suggestion.

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]
Approve after @get-itips suggestion.

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Comment by get-itips** [comment]
> @Kateyanne Please merge this PR. Thanks

A suggestion review by the PR author is still pending.

**Pull request merged**